src/Containers/VendorMarketList/VendorMarketList.scss

```diff
@@ -5,10 +5,16 @@
   flex-direction: column;
   align-items: center;
   justify-content: flex-start;
-  background-color: white;
-  border: 3px solid black;
+  background-color: rgba(0,0,0,0.4);
+  border: 2px solid khaki;
   border-radius: 20px;
-  background-color: #F2F9F9;
+  // background-color: #F2F9F9;
```

Comment:
I think that something might have gone wrong here - it looks like the background and text are the same color so you can not see the text at all. even on hover you can only barely see it. The remove and Add buttons look pretty funky with that as well now that they do not match the styling